cypress/e2e/timeline/timeline.spec.ts

```diff
@@ -453,6 +453,118 @@ describe("Timeline", () => {
             cy.get(".mx_RoomView_searchResultsPanel").percySnapshotElement("Highlighted search results");
         });
 
+        it("should find words in various languages and highlight them on search result", () => {
+            // "Test" in various languages
+            const stringAr = "اِمْتِحَان‎";
+            const stringEl = "εξέταση";
+            const stringHe = "מִבְחָן";
+            const stringHi = "आज़माइश";
+            const stringJa = "テスト";
+            const stringKo = "테스트";
+            const stringTh = "การสอบ";
+            const stringZh = "测试";
+
+            const MESSAGE = "Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor";
+
+            const sendString = (string: string) => {
+                cy.sendEvent(roomId, null, "m.room.message" as EventType, {
+                    msgtype: "m.text" as MsgType,
+                    body: string,
+                });
+            };
+
+            const searchAndHighlight = (string: string) => {
+                cy.get(".mx_SearchBar_input input").clear().invoke("val", string).trigger("input").type("{enter}");
+                cy.get(".mx_EventTile:not(.mx_EventTile_contextual) .mx_EventTile_searchHighlight").should("exist");
+            };
+
+            // Open roomId room
+            cy.visit("/#/room/" + roomId);
+
+            // Wait until configuration is finished
+            cy.contains(
+                ".mx_RoomView_body .mx_GenericEventListSummary .mx_GenericEventListSummary_summary",
+                "created and configured the room.",
+            ).should("exist");
```

Comment:
I feel like we should move this out to a separate function rather than repeating it for every single test.

Reply:
This is not the scope of this PR.